### PR TITLE
[CSFS-13] Newsletter Form Validation

### PR DESCRIPTION
**Pull request by DamisiM**
## Task: `[CSFS-13] Newsletter Form Validation`
- `Dom task`

### CODE REVIEW

File: submissions/damisiMahmood/HTML/productPage.html [hunk 1]
```diff
@@ -69,10 +69,14 @@ <h2>RELATED PRODUCTS</h2>
                 <li><a href="#">Terms and Conditions</a></li>
             </ul>
         </div>
-        <div class="middle-footer"> 
-            <p> Sign up for our Newsletter </p>
-			<input type="email" name="email" id="email" placeholder="Enter your email address"><button><span>SUBMIT</span></button>
-        </div>
+        <form id="validateNews">
```
Comment (ToluAkin) [inline submissions/damisiMahmood/HTML/productPage.html:72]: @DamisiM Why did you wrap this in a form?

File: submissions/damisiMahmood/HTML/productPage.html [hunk 2]
```diff
@@ -88,5 +92,7 @@ <h2>RELATED PRODUCTS</h2>
             <p>Copyright 2020 i-damselseliteshopping.com 2020. All rights reserved.</p>
         </div>
     </footer>
+    <script src="../domManipulation/newsletter.js"></script>
```
Comment (ToluAkin) [inline submissions/damisiMahmood/HTML/productPage.html:95]: @DamisiM Why are you adding the same file twice?

File: submissions/damisiMahmood/domManipulation/newsletter.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+let newsletter = document.getElementById('validateNews');
+let newsEmail = document.getElementById('newsemail');
+let newsEmailRegex = /^\w+([-]?\w+)*@\w+([-]?\w+)*(.\w{3,4})+$/;
+
+
+function newsletterValidation(event) {
+    if (newsEmail.value === "") {
+
```
Comment (ToluAkin) [inline submissions/damisiMahmood/domManipulation/newsletter.js:8]: Remove extra space @DamisiM

File: submissions/damisiMahmood/domManipulation/newsletter.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+let newsletter = document.getElementById('validateNews');
```
Comment (ToluAkin) [inline submissions/damisiMahmood/domManipulation/newsletter.js:1]: @DamisiM You could just target the class `middle-footer` instead

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/damisiMahmood/htmlForms/signup.html [hunk 5]
```diff
@@ -96,6 +100,7 @@ <h1><span> Register your Account</span></h1>
             </div>
         </footer>
         <script src="../domManipulation/signUp.js"></script>
-        </body>
-    </html>
+        <script src="../domManipulation/newsletter.js"></script>
```
Comment (adaugochi) [inline submissions/damisiMahmood/htmlForms/signup.html:103]: Why are you adding the same file twice? @DamisiM

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**